src/domain/synthetics/markets/createDepositTxn.ts

```diff
@@ -133,7 +133,8 @@ export async function createDepositTxn(chainId: number, signer: Signer, p: Creat
 }
 
 interface CreateGlvDepositParams extends CreateDepositParams {
-  market: string;
+  glvAddress: string;
+  marketTokenAmount: bigint;
   isMarketTokenDeposit: boolean;
 }
 
```

Comment:
I think it makes sense to move `createGlvDepositTxn` to a separate file. We have file per each tx type, and I think this is a good strategy to follow